docs/src/manifolds/metric.md

```diff
@@ -1,7 +1,46 @@
 # Metric manifold
 
+A Riemannian manifold always consists of a [topological manifold](https://en.wikipedia.org/wiki/Topological_manifold) together with a smoothly varying metric $g$.
+
+However, often there is a implicitly assumed (daefault) metric, like the usual
+inner product on [`Euclidean`](@ref) space. This decorator takes this into
+account. Hence it is not necessary to acts as a (as transparent as possible)
+decorator, see [`is_decorator_manifold`](@ref) for details.
+
+This manifold servers two purposes: To implement different metrics (i.g. in closed
+form) for one [`Manifold`](@ref) and to provide a way to compute geodesics
+on manifolds, where this [`Metric`](@ref) does not yield closed formula. 
```

Comment:
I think eventually all other functions would also have a default implementation based on the metric, right? Like `exp`, `log`, parallel transport etc.

Reply:
Well I hope so, yes, for now there is only exp implemented based on the metric tensor. I have not yet read how to do the remaining ones numerically – but also since my goal is more to implement different metrics